### PR TITLE
tests: bring `deno` fixture template in line with actual template

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: integration/helpers/deno-template/.eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (kentcdodds) [inline integration/helpers/deno-template/.eslintignore:1]: Could we add a comment explaining why this file is here?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Looks good. Just one thing.

File: integration/helpers/deno-template/.eslintignore (hunk 1 above)
Comment (pcattori) [reply integration/helpers/deno-template/.eslintignore:1]: Yea I think we can just remove this file and be ok.

File: integration/helpers/deno-template/package.json [hunk 2]
```diff
@@ -2,14 +2,15 @@
   "name": "remix-template-deno",
   "private": true,
   "sideEffects": false,
+  "scripts": {},
```
Comment (pcattori) [inline integration/helpers/deno-template/package.json:5]: Why is `scripts` needed?

Comment (MichaelDeBoey) [reply integration/helpers/deno-template/package.json]: Leftover from cleanup, will remove

**Review by kentcdodds — APPROVED** [review]
👍

**Pull request merged**